src/test/java/org/fundacionjala/coding/Marco/SortInnerContentTest.java

```diff
@@ -0,0 +1,58 @@
+package org.fundacionjala.coding.Marco;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+/**
+ * This kata was made by marco mendez august 29, 2017.
+ */
+public class SortInnerContentTest {
+    /**
+     * Test example 1.
+     */
+    @Test
+    public void testSortInnerContent1() {
+        SortInnerContent sort = new SortInnerContent();
```

Comment:
You are repeating this code in all your @Test methods
please move it to a @Before method.